types/sequencify/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for sequencify 0.0
```

Comment:
I can't guarantee that the `@types` publishing script won't choke on this. Can you change to `0.1`?